### DISCUSSION TITLE
Education version - version without age restriction

### DISCUSSION THREAD

**Discussion by NiTRoeSE** [element-hq/element-meta#2043] [Ideas]
Hello, we use matrix + element in a education environment with own servers without federation.
All channels we have are moderated.

Unfortunately, Element is age-restricted so 90%+ of all students can't even install the app.

On the one hand, this is a pity for us and on the other hand, it is not conducive to the spread and acceptance of matrix among users.
In the end, this does not only affect our concern.
Basically, it would be great if matrix were a really serious competitor to services such as WhatsApp, etc.

But for that to happen, it would be necessary to have mobile apps that younger users can install.
Ultimately, it is precisely this user base that determines distribution and acceptance.

I think it would be a win to win situation for everyone involved.

As far as I know, the reason for the age classification is the possibility to enter guest rooms on home servers like matrix.org that are not moderated.

Maybe there is a way to create an education app version that doesn't have this feature and gets a better age rating in Playstore and Apple's Appstore.

In this way, element can also become a concept among young people, which ideally already sets the direction for their later digital lives. In the best case, questionable services like WhatsApp or Telegram will shrink and Matrix + Element grow up.

**How would you like to achieve it?**

Creating a education version or a "Element Kids" version without the function to enter a public room without moderation as a guest user.

**Have you considered any alternatives?**

Element X is at the moment rated 0+ in Playstore because it lacks the feature which leads to higher rating.
That's great at the moment, but i think its just a question of time..

Unfortunately I don't know Android or IOS development and also I don't know Swift and Kolin languages.
So i can't help in development.

**Comment by aaronraimist** [comment]
> As far as I know, the reason for the age classification is the possibility to enter guest rooms on home servers like matrix.org that are not moderated.

I don't know if that is the reason. On iOS, most other messaging apps have a lower age listed then Element's 17+ age rating. Signal, Snapchat, Skype, Facebook Messenger, WhatsApp are all listed as 12+ years old on iOS. Slack, Google Meet, Google Chat, Microsoft Teams, and IRCCloud are even listed as 4+ years old.

At least on iOS, I think the reason may be that they have said the app allows unrestricted web access (the app doesn't actually allow this AFAIK). Any time the developer says their app allows unrestricted web access, the age limit is automatically set to 17+. https://github.com/vector-im/element-ios/issues/2097. ~~I don't know that for sure that is the reason but that is my guess.~~ Actually no this is not a guess, it is the reason. You can see on the App Store that the reason for the 17+ rating is that Element has said the app allows for unrestricted web access.
![screenshot](https://github.com/vector-im/element-meta/assets/5855073/c1e7c1f8-26ff-4d14-b0f6-877c99620bcb)

This is what the screen looks like in App Store Connect where you determine the age rating. How I have it set in the screenshots would give a 12+ rating. Some of these like "Profanity or Crude Humor" and "Cartoon or Fantasy Violence" can even be set to Frequent/Intense and it will still give the app a 12+ age rating.
![screenshot 1](https://github.com/vector-im/element-meta/assets/5855073/57577740-4cf9-48cc-9312-34e9deefbca5)
![screenshot 2](https://github.com/vector-im/element-meta/assets/5855073/d32486fe-6b86-42d0-81e6-0f042d72b3ed)
![screenshot 3](https://github.com/vector-im/element-meta/assets/5855073/8ac57006-0f4c-495b-b996-ccff710765d2)

**Reply by NiTRoeSE** [reply]
So in that case there is a good chance to get the apps rated as 12+.
That would be great.

Did someone know how it is in play store ?

**Comment by cooklillian** [comment]
<p>A version of education without age restrictions sounds like a fantastic idea, as it allows individuals of all ages to pursue knowledge and personal growth at their own pace. This inclusive approach can break down barriers and foster a lifelong love for learning.Removing age restrictions in education versions&nbsp;<span data-sheets-value="{&quot;1&quot;:2,&quot;2&quot;:&quot;https://researchpaperwriter.org/custom-research-paper-writing-service&quot;}" data-sheets-userformat="{&quot;2&quot;:268416,&quot;10&quot;:2,&quot;14&quot;:{&quot;1&quot;:2,&quot;2&quot;:1136076},&quot;15&quot;:&quot;Arial&quot;,&quot;21&quot;:1}" data-sheets-hyperlink="https://researchpaperwriter.org/custom-research-paper-writing-service" data-sheets-hyperlinkruns="{&quot;1&quot;:0,&quot;2&quot;:&quot;https://researchpaperwriter.org/custom-research-paper-writing-service&quot;}"><a class="in-cell-link" href="https://researchpaperwriter.org/custom-research-paper-writing-service" target="_blank" rel="noopener">https://researchpaperwriter.org/custom-research-paper-writing-service</a></span> could promote a more diverse and inclusive learning environment, where people from various age groups can share their unique perspectives and experiences, enriching the overall educational experience for everyone involved. It's a step towards creating a truly accessible and equitable learning system.</p>

**Comment by NiTRoeSE** [comment]
Following Fork of Element (SchildiChat) is also rated 0+ in Playstore: https://play.google.com/store/apps/details?id=de.spiritcroc.riotx

<img width="1417" alt="Bildschirmfoto 2023-10-09 um 10 42 34" src="https://github.com/vector-im/element-meta/assets/24574743/ac5062f3-a90e-4e13-b718-605fbf1f1669">

I simply don't understand why that's the case and Element is 18+ rated.
Can please someone recheck the rating process for iOS and Android so that Element can be available for a much wider range of users. ( 12+ is fine )

Thanks in Advanced
